internal/services/netapp/netapp_volume_resource.go

```diff
@@ -474,6 +479,8 @@ func resourceNetAppVolumeCreate(d *pluginsdk.ResourceData, meta interface{}) err
 		avsDataStoreEnabled = volumes.AvsDataStoreEnabled
 	}
 
+	isLargeVolume := d.Get("is_large_volume").(bool)
+
```

Comment:
We also will need deeper validations, there are many that can't be done on schema but must be done here and return error as you see other validations above in this method.

Here are a few from the documentation that must be done here (https://learn.microsoft.com/en-us/azure/azure-netapp-files/large-volumes-requirements-considerations#requirements-and-considerations):

Note that other requirements are not listed below because the updates are not made yet in Terraform so they won't apply at this point.

* Large volumes aren't currently supported with cross-region replication.
* Large volumes aren't currently supported with cross-zone replication.